src/SymbolDirectories/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(SymbolDirectories)
+add_library(SymbolDirectories STATIC)
+
+target_compile_options(SymbolDirectories PRIVATE ${STRICT_COMPILE_FLAGS})
+target_include_directories(SymbolDirectories PUBLIC include/)
+target_link_libraries(SymbolDirectories PUBLIC
+                                        Qt5::Core
+                                        Qt5::Test)
```

Comment:
As far as I can see, Qt5::Test is not needed.

Reply:
true, done